README.md

```diff
@@ -10,6 +10,10 @@
   <a href="https://opensource.org/licenses/Apache-2.0"><img src="https://img.shields.io/badge/License-Apache_2.0-blue.svg"></a>
 </p>
 
+## Intro video
```

Comment:
hi @MarioUhrik, did we think about centering this one, too?

ref
![image](https://user-images.githubusercontent.com/61180606/220601959-5dc6956e-f2e9-4827-9fa1-b0980ca2f129.png)